pkg/fs/apkfs.go

```diff
@@ -235,8 +235,18 @@ func (a *APKFS) Stat(path string) (fs.FileInfo, error) {
 	}
 	return &apkFSFileInfo{file: file, name: file.name[strings.LastIndex(file.name, "/"):]}, nil
 }
-
+func correctApkFSPath(path string) string {
+	if path == "." {
+		path = "/"
+	}
+	if len(path) > 3 && path[:2] == "./" {
+		path = path[1:]
+	}
+	path = filepath.Clean(path)
```

Comment:
```suggestion
```

Move that.

Reply:
Moving it before the other checks breaks the path resolution